public/javascripts/ng-apps.js

```diff
@@ -41,6 +41,23 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
         'content': {
           templateUrl: 'views/content/student.html',
           controller: 'studentController'
+        },
+        'studentView@index.student': {
+          templateUrl: 'views/content/studentView.html'
+        }
+      }
+    })
+    .state('index.student.addEntry', {
```

Comment:
Ah okay. Because you can't name two states the same thing regardless of
parent, I'm fine with ur change. I can either do it this afternoon or you
can do it and I'll thumbs it up

On Mon, Apr 11, 2016 at 8:56 AM Nora Mohamed <notifications@github.com>
wrote:

> In public/javascripts/ng-apps.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/80#discussion_r59198539>
> :
>
> > @@ -41,6 +41,23 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
> >          'content': {
> >            templateUrl: 'views/content/student.html',
> >            controller: 'studentController'
> > +        },
> > +        'studentView@index.student': {
> > +          templateUrl: 'views/content/studentView.html'
> > +        }
> > +      }
> > +    })
> > +    .state('index.student.addEntry', {
>
> https://github.com/angular-ui/ui-router/wiki/Nested-States-and-Nested-Views
> hmm not exactly - there aren't any official examples of double nested views
> i suppose you could also just do child.grandchild:{parent: 'parent',
> templateURL...}
> the name just feels very long and unnecessary, but we can stay with it
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/80/files/82c4e851bb880cbb3a20ef88419c47beea1ab840#r59198539>
>